schemas/ohi-jmx.yml

```diff
@@ -0,0 +1,74 @@
+---
+- src: "{app_name}_linux_{version}_{arch}.tar.gz"
+  uploads:
+    - type: file
+      dest: "{dest_prefix}binaries/linux/{arch}/{src}"
+  arch:
+    - amd64
+    - 386
+    - arm
+    - arm64
+
+- src: "{app_name}-{arch}.{version}.zip"
+  uploads:
+    - type: file
+      dest: "{dest_prefix}binaries/windows/{arch}/{src}"
+  arch:
+    - amd64
+    - 386
+
+# Windows installers are .exe and have the '-installer' string in the name for integrations bundling nrjmx.
+- src: "{app_name}-amd64-installer.{version}.exe"
+  uploads:
+    - type: file
+      dest: "{dest_prefix}windows/integrations/{app_name}/{src}"
+    - type: file
+      override: true
+      dest: "{dest_prefix}windows/integrations/{app_name}/{app_name}-amd64-installer.exe"
+
+- src: "{app_name}-386-installer.{version}.exe"
+  uploads:
+    - type: file
+      dest: "{dest_prefix}windows/386/integrations/{app_name}/{src}"
+    - type: file
+      override: true
+      dest: "{dest_prefix}windows/386/integrations/{app_name}/{app_name}-386-installer.exe"
```

Comment:
IIRC jmx integrations did not have a 386 installer (due to JMX not being available for 386)